BuildAndTest.cmd

```diff
@@ -64,60 +64,63 @@ if "%ERRORLEVEL%" NEQ "0" (
 goto ExitFailed
 )
 
-msbuild /verbosity:minimal /target:rebuild src\Everything.sln /p:"Configuration=%Configuration%" /p:"Platform=Any CPU" /filelogger /fileloggerparameters:Verbosity=detailed /p:"RunBinSkim=false" /p:"BinSkimVerboseOutput=true"
+msbuild /verbosity:minimal /target:rebuild src\Everything.sln /filelogger /fileloggerparameters:Verbosity=detailed /p:"RunBinSkim=false" /p:"BinSkimVerboseOutput=true"
+set Platform=AnyCPU
```

Comment:
Have to reset platform to 'AnyCPU' without a space after MSBUILD because MSBuild will remove the space from 'Any CPU' but dotnet pack does not.  This was causing the code to build twice during BuildAndTest.  MSBuild would be built to 'AnyCPU_Release' and dotnet pack and dotnet xunit would look for their source in 'Any CPU_Release'

Reply:
Also, passing 'AnyCPU' without a space, to MSBuild, results in an error.  This is why it has to start off as 'Any CPU' and change to 'AnyCPU' after MSBuild

---
In reply to: [145827547](https://github.com/Microsoft/sarif-sdk/pull/639#discussion_r145827547) [](ancestors = 145827547)

Reply:
That's horrible ;-)